easybuild/easyconfigs/n/numpy/numpy-1.17.2-CrayGNU-19.10.eb

```
@@ -3,13 +3,6 @@ easyblock = 'PythonPackage'
 name = 'numpy'
 version = '1.17.2'
 
-local_py_maj_ver = '3'
-local_py_min_ver = '6'
-local_py_rev_ver = '5.7'
-
-local_pyver = '%s.%s.%s' % (local_py_maj_ver, local_py_min_ver, local_py_rev_ver)
-local_pyshortver = '%s.%s' % (local_py_maj_ver, local_py_min_ver)
-
 homepage = 'https://github.com/numpy/numpy'
 description = "Horovod is a distributed training framework for TensorFlow."
```

a bit unrelated but as we are at it...
```suggestion
description = """NumPy is the fundamental package for scientific computing with Python. It contains among other things:
 a powerful N-dimensional array object, sophisticated (broadcasting) functions, tools for integrating C/C++ and Fortran
 code, useful linear algebra, Fourier transform, and random number capabilities. Besides its obvious scientific uses,
 NumPy can also be used as an efficient multi-dimensional container of generic data. Arbitrary data-types can be
 defined. This allows NumPy to seamlessly and speedily integrate with a wide variety of databases."""
```

---

thanks @henrique !